src/injections/js/bug1818818-fastclick-legacy-shim.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+/**
+ * Bug 1818818 - Neutralize FastClick
+ *
+ * The patch is applied on sites using older version of FastClick library.
+ * This allows to disable FastClick and fix various breakage caused
+ * by the library.
+ **/
+
+/* globals exportFunction */
+
+const proto = CSS2Properties.prototype.wrappedJSObject;
+Object.defineProperty(proto, "msTouchAction", {
+  get: exportFunction(function() {
+    return "none";
+  }, window),
+
+  set: exportFunction(function() {}, window),
+});
```

Comment:
This is simpler variation of a fastclick patch for an older version of fastclick that don't use `touchAction` but only `msTouchAction` (https://bugzilla.mozilla.org/show_bug.cgi?id=1818818#c0)